android/src/main/AndroidManifest.xml

```diff
@@ -1,10 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-  package="io.chirp.chirpsdk">
-
-    <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS" />
```

Comment:
How are we handling permissions requests now?

Reply:
I see, you moved these to the app, right?

Reply:
Yes, I think I checked with the documentation and they were inside the `<application>` tag there